pygments/lexers/c_like.py

```diff
@@ -663,3 +649,56 @@ class OmgIdlLexer(CLexer):
             include('annotation_appl'),
         ],
     }
+class FlexLexer(CLexer):
+    """
+    Lexer for Flex <http://dinosaur.compilertools.net/flex/index.html>, 
+    A fast scanner generator.
+    .. versionadded:: 2.5
```

Comment:
```suggestion

    .. versionadded:: 2.5
```

Here you need an empty line, or the directive block won't be recognized.

You can check the output by running `make html` in the `doc/` directory and opening `_build/html/docs/lexers.html`.

Reply:
also it would be 2.13 now.